inf-clojure.el

```diff
@@ -773,7 +787,7 @@ prefix argument PROMPT-FOR-NS, it prompts for a namespace name."
   (let ((ns (if prompt-for-ns
                 (car (inf-clojure-symprompt "Ns vars" (clojure-find-ns)))
               (clojure-find-ns))))
-    (comint-proc-query (inf-clojure-proc) (format inf-clojure-ns-vars-form ns))))
+    (inf-clojure--send-string (inf-clojure-proc) (format (inf-clojure-ns-vars-form) ns))))
```

Comment:
I just wonder what's the difference between `send-string` and `proc-query`. I said I don't think it's needed as likely setting the ns is not going to be first thing you do with the repl, but I might be mistaken in my assumption.